libraries/Microsoft.Bot.Builder.AI.Orchestrator/Schemas/Microsoft.OrchestratorRecognizer.schema

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": "implements(Microsoft.IRecognizer)",
+    "title": "Orchestrator Recognizer",
+    "description": "Orchestrator recognizer.",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "title": "Id",
+            "description": "Optional unique id using with RecognizerSet."
+        },
+        "modelPath": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "Model",
+            "description": "NLR model file path.",
+            "default": "=settings.orchestrator.modelpath"
+        },
+        "snapshotPath": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "Endpoint Key",
+            "description": "SnapShot file path.",
+            "default": "=settings.orchestrator.shapshotpath"
+        },
+        "entityRecognizers": {
+            "type": "array",
+            "items": {
+                "$kind": "Microsoft.IEntityRecognizer"
```

Comment:
>IEntityRecognizer [](start = 36, length = 17)

Is there any case where you would want to get entities from a LUIS recognizer?  This only supports the standalone entities.

Reply:
Leaving this as is. If customer wants to use entities from LUIS recognizer, then they can always use a recongnizer set with orchestratorRecognizer and a LuisRecognizer.